src/pack/ui_func.py

```diff
@@ -0,0 +1,41 @@
+from .exceptions import valid_argument
+from .exceptions import valid_sign
+from .func import division_of_arg
+from .func import mult_of_arg
+from .func import subtraction_of_arg
+from .func import sum_of_arg
+
+
+def menu():
+    while True:
+        a = input("enter first argument ")
+        a = valid_argument(a)
```

Comment:
```suggestion
        a = get_valid_int_argument()
```

Пускай функция get_valid_int_argument ничего не принимает и всю логику чтения с консоли делает внтутри

Reply:
Так у меня же так и было))